dependencies.yaml

```diff
@@ -377,6 +378,33 @@ dependencies:
               - *libcusolver114
               - *libcusparse_dev114
               - *libcusparse114
+  cuda_wheels:
+    specific:
+      - output_types: pyproject
+        matrices:
+          - matrix:
+              cuda: "12.*"
+              cuda_suffixed: "true"
```

Comment:
```suggestion
```

I don't think we need the `cuda_suffixed` mechanism here at all.

That was added as part of https://github.com/rapidsai/build-planning/issues/31 to account for the fact that in some builds (DLFW, pip devcontainers), we build a bunch of libraries from source and prefer to not append CUDA suffixes to their distribution names for simplicity in scripts.

That doesn't apply to this set of dependencies, where for DLFW / pip devcontainers we simply don't want them in the wheel dependency list at all.